RELEASE_NOTES.md

```diff
@@ -2,6 +2,8 @@
 
 ## New in git-machete 3.11.2
 
+- fixed: `git machete` now correctly infers remote for fetching of branch when the branch is associated with more than one remote
+- fixed: `git machete` now correctly infers local branch's remote counterpart for fetching of branch when the branch is associated with more than one remote
```

Comment:
These two are technically a different thing... but I'd just leave the first point; such level of detail isn't need for release notes